banca-noix/src/main/resources/config/liquibase/changelogs/202109152147_criando_usuario.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext
+                    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd
+                    http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+
+    <changeSet id="202109152147" author="matheus.loose">
+        <createTable tableName="USUARIO">
+            <column name="ID" type="bigint" autoIncrement="true" remarks="Identificador(ID) para usuario">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="NOME" type="varchar(100)" remarks="Nome do usuario">
+                <constraints nullable="false"/>
+            </column>
+            <column name="EMAIL" type="varchar(50)" remarks="Email unico do usuario">
+                <constraints unique="true" nullable="false"/>
+            </column>
+            <column name="CPF" type="varchar(15)" remarks="CPF unico do usuario">
+                <constraints unique="true" nullable="false"/>
+            </column>
+            <column name="DT_NASCIMENTO" type="date" remarks="Data de Nascimento do usuario">
+                <constraints nullable="false"/>
+            </column>
+            <column name="TELEFONE" type="varchar(14)" remarks="Telefone para contato com o usuario">
+                <constraints nullable="false"/>
+            </column>
+            <column name="ST_ATIVIDADE" type="boolean" remarks="Status de atividade do usuario na empresa">
+                <constraints nullable="false"/>
+            </column>
+            <column name="CARGO_ID" type="bigint" remarks="Referencia de identificação ao cargo do usuario">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+
+    <changeSet id="202109161010" author="matheus.loose">
+        <addForeignKeyConstraint baseTableName="USUARIO"
+                                 baseColumnNames="CARGO_ID"
```

Comment:
mais atenção galera, vcs corrigiram o comentario da linha de cima, mas tem o mesmo erro algumas linhas  a baixo, quando um erro for apontado ele deve ser corrigido em todos os arquivos